spec/features/saml/ial1_sso_spec.rb

```diff
@@ -202,7 +195,15 @@
   context 'requesting verified_at for an IAL1 account' do
     it 'shows verified_at as a requested attribute, even if blank' do
       user = create(:user, :signed_up)
-      saml_authn_request = auth_request.create(ial1_with_verified_at_saml_settings)
+      saml_authn_request = saml_authn_request_url(
+        saml_overrides: {
+          issuer: 'saml_sp_ial2',
```

Comment:
Previously this was `'https://rp1.serviceprovider.com/auth/saml/metadata'`. Should it just be...

```suggestion
          issuer: sp1_issuer,
```

Reply:
great catch!